wsmaster/che-core-api-devfile/src/main/java/org/eclipse/che/api/devfile/server/convert/component/editor/EditorComponentProvisioner.java

```diff
@@ -50,7 +52,10 @@ public void provision(DevfileImpl devfile, WorkspaceConfigImpl workspaceConfig)
         workspaceConfig
             .getAttributes()
             .getOrDefault(EDITOR_COMPONENT_ALIAS_WORKSPACE_ATTRIBUTE, editorAttribute));
-
+    editorComponent.setMemoryLimit(
+        workspaceConfig
+            .getAttributes()
+            .get(format(SIDECAR_MEMORY_LIMIT_ATTR_TEMPLATE, editorAttribute.split(":")[0])));
```

Comment:
What if this attribute is missing? Will this break devfile?

Reply:
Nope, this field isn't required, so it can be `null` as well

Reply:
Wouldn't this produce IllegalArgumentException in MachineResolver?